data_service/src/match_processing/match_parser.rs

```diff
@@ -0,0 +1,34 @@
+use riven::models::match_v5::Match;
+
+use crate::types::PuuidToChampionMapping;
+
+pub async fn parse_match(match_data: Match) -> Vec<PuuidToChampionMapping> {
+    // Get match data.
```

Comment:
TODO(pyctal): Implement this function to get the test below to pass

Reply:
On it